lib/configuration/variables/resolve.js

```diff
@@ -483,6 +483,29 @@ Object.defineProperties(
               propertyPath,
               ensureArray(dependencyPropertyPathKeys, { ensureItem: ensureString })
             ),
+          resolveVariable: async (variableString) => {
+            variableString = `\${${ensureString(variableString, {
+              Error: ServerlessError,
+              name: 'variableString',
```

Comment:
Let's add `code` to `ServerlessError` via `errorCode`

Reply:
Good point, addressed 👍